app/bundles/PluginBundle/Form/Type/CompanyFieldsType.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/*
+ * @copyright   2014 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\PluginBundle\Form\Type;
+
+use Mautic\CoreBundle\Helper\InputHelper;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Class SocialMediaServiceType.
+ */
+class CompanyFieldsType extends AbstractType
+{
+    /**
+     * @param FormBuilderInterface $builder
+     * @param array                $options
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        foreach ($options['integration_company_fields'] as $field => $details) {
+            $label = (is_array($details)) ? $field : $details;
```

Comment:
Should this be `$label = (is_array($details)) ? $details['label'] : $details;` so that it shows the label rather than Salesforce field name?